staging/src/k8s.io/apiserver/pkg/admission/metrics/metrics.go

```diff
@@ -217,13 +218,24 @@ func newAdmissionMetrics() *AdmissionMetrics {
 		},
 		[]string{"name", "type", "operation", "code", "rejected"})
 
+	matchConditionEvalError := metrics.NewCounterVec(
+		&metrics.CounterOpts{
+			Namespace:      namespace,
+			Subsystem:      subsystem,
+			Name:           "admission_match_condition_evaluation_errors_total",
+			Help:           "Admission match condition evaluation errors count, identified by name of resource containing the match condition and broken out for each admission type (validating or mutating).",
+			StabilityLevel: metrics.ALPHA,
+		},
+		[]string{"name", "type"})
```

Comment:
Is it possible to distinguish error types? Specifically, runtime budget exhaustion vs. other error types would be interesting.

Reply:
We might also want to include outcome as a dimension to this. Maybe something like this (see if you can come up with better names...):

`outcome`:
- `policyFail`: when failurePolicy = fail
- `policyIgnore`: when failurePolicy = ignore
- `shortCircuitSkip`: when an expression returns `false`

Reply:
Don't block the PR on this, we can add it later.